src/assets/css/main.css

```
@@ -179,3 +179,13 @@ canvas {
 .sidebar .sidebar-background {
   background-color: #00000080!important;
 }
+
+/*
+ * add for
+ * @oruga-ui/theme-bulma”: “^0.4.1"
+ * text color of success events
+ * ref: PhoneLogin.vue, PhoneEntry.vue
+ */
+.help.is-success {
+  color: #16a34aff!important;
```

```suggestion
  color: #16a34aff;
```

こちらは !important flag を付けないとスタイルは当たりませんか？

上記のように !important flag を付けなくても良いのであれば、付けないのが望ましいです・・・

---

飽く迄、個人的な意見です・・・

---

!importantがないと上書きされないので修正できないケースでつけています！！

---

importantはつけなくて良いならつけないほうが良いのは同意です。

---

なるほど。ありがとうございます！